src/tools/miri/tests/pass/track-alloc-1.stderr

```diff
@@ -1,5 +0,0 @@
-note: tracking was triggered
-   |
-   = note: created extern static allocation of SIZE bytes (alignment ALIGN bytes) with id 1
-   = note: (no span available)
```

Comment:
I think this changes what the test does, but it's also hard to fix.

@rust-lang/miri should we try some other very low alloc ID and hope it's in the init code, or just ditch this test?

Reply:
I think just ditching the test seems best. This will keep coming back over and over again when something changes in rustc.